packages/desktop-gui/cypress/integration/specs_list_spec.js

```diff
@@ -435,13 +455,26 @@ describe('Specs List', function () {
 
           cy.get('.specs-list .file')
           .should('have.length', this.numSpecs)
+
+          cy.contains('.all-tests', 'Run all specs')
         })
 
         it('shows empty message if no results', function () {
           cy.get('.filter').clear().type('foobarbaz')
           cy.get('.specs-list').should('not.exist')
 
           cy.get('.empty-well').should('contain', 'No specs match your search: "foobarbaz"')
+
+          cy.contains('.all-tests', 'No specs')
+        })
+
+        it('disables run all tests if no results', function () {
+          cy.get('.filter').clear().type('foobarbaz')
+
+          cy.contains('.all-tests', 'No specs').should('have.class', 'disabled').click()
```

Comment:
If this was a proper button, Cypress wouldn't let you click on it.

Reply:
yeah, I asked why we are using `<a>` and not a button in the test runner channel, no response, but this should be a button